sbc_compassion/tests/test_letter_import.py

```diff
@@ -0,0 +1,117 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp.tests import common
+
+import base64
+import csv
+import os
+import re
+
+IMPORT_DIR = os.path.join(os.path.dirname(__file__), "testdata/import")
+
+
+def get_file_content(path):
```

Comment:
Strange github has erased my previous comment about moving this function in tools/import_letters_functions.py